lms/djangoapps/edxnotes/tests.py

```diff
@@ -1077,7 +1077,10 @@ def test_get_id_token(self):
         response = self.client.get(self.get_token_url)
         assert response.status_code == 200
         client = Application.objects.get(name='edx-notes')
-        jwt.decode(response.content, client.client_secret, audience=client.client_id)
+        jwt.decode(
+            response.content, client.client_secret,
+            audience=client.client_id, algorithms=settings.JWT_AUTH['JWT_ALGORITHM']
```

Comment:
Its already in use here in platform
https://github.com/edx/edx-platform/blob/3152c14192dcccac79b7b7a76eba91b380eba78a/openedx/core/djangoapps/oauth_dispatch/tests/mixins.py#L46